README.md

```diff
@@ -1,14 +1,19 @@
 # Get started
+
 ## In a Container
+
 1. Install docker
-2. `docker compose up --build`
+2. change to docker directory `cd docker`
```

Comment:
just curious, is there a reason for moving it into a separate dir?

Reply:
no functional reason, but after researching a bit, it made more sense, especially if we are going to create more `Dockerfiles` or have different `compose` definitions.

eg, if we wanted to create a second service (like for GS polling), we would need to create another `Dockerfile`, and that would make the root dir pretty messy if we have `Dockerfile.api` and `Dockerfile.polling` etc.

Reply:
ok sounds good